### DISCUSSION TITLE
Integrate with LMS Learndash

### DISCUSSION THREAD

**Discussion by minhcasi** [inspireui/support#5067] [New Feature/Ideas Requests]
compatible with LMS plugin like learndash

**Comment by kodzodeku** [comment]
I m looking out for this. This would be a very useful integration and I dare to say it is even an urgent one. Please, put this on top of the list. It is a much needed and long awaited feature. 
Thanks.

**Comment by ameenbadri7** [comment]
Fluxstore as LMS platform .. Wow

**Comment by minhcasi** [comment]
Here is another LMS -
https://www.themeum.com/product/tutor-lms/

**Comment by davidom0101** [comment]
+1 For Tutor LMS integration

**Comment by Seofyro** [comment]
+1

**Reply by Seofyro** [reply]
LEARNDASH
